google/cloud/apigee/registry/applications/v1/diff_analytics.proto

```diff
@@ -0,0 +1,23 @@
+syntax = "proto3";
+package google.cloud.apigee.registry.applications.v1;
+// [START go_declaration]
+option go_package = "github.com/apigee/registry/rpc;rpc";
+// [END go_declaration]
+
+/* This proto contains the diff of a spec and its revision. Changes that are
+additions fall in the added field. Changes that are Deletions fall in the
+deleted field, and changes that modify values fall in the value_modify map
+field. */
+message Diff {
+  // Change will be displayed in format foo.bar.x
+  repeated string additions = 1;
+  // Change will be displayed in format foo.bar.x
+  repeated string deletions = 2;
+
+  message ValueChange {
+    string From = 1;
+    string To = 2;
+  }
+  // Change will be displayed in format foo.bar.x {to.x from.x}
```

Comment:
I don't think we should document how this message "will be displayed" since the code that displays it may change without knowing to update this comment. We might decide to serialize ValueChange using `:` to separate its fields from its values instead of `.`, and shouldn't need to update this documentation. 

This comment should stick to explaining _what_ is included in the map and the intended format of the string key. The `ValueChange` message can have its own documentation explaining the meaning of the `to` and `from` fields.